src/test/rustdoc-gui/toggle-click-deadspace.goml

```diff
@@ -0,0 +1,10 @@
+// This test ensures that clicking on a method summary, but not on the "[-]",
+// doesn't toggle the <details>.
+debug: true
+goto: file://|DOC_PATH|/test_docs/struct.Foo.html
+size: (1000, 1000)
+assert-attribute: (".impl-items .rustdoc-toggle", {"open": ""})
+click: (249, 509) // This is the position of "pub" in "pub fn a_method"
```

Comment:
You can click directly on `h4.code-header`. I'd much prefer avoiding to use hard coded positions if possible. :)